test/fixtures/packages/invalid-package.json

```diff
@@ -1,21 +1,21 @@
-{
-  "name": "test-package",
-  "description": "Test Package for Testing!",
-  "version": "0.0.1",
-  "tags": [
-  ],
-  "files": [
-  ],
-  "dependencies": {
-    "gulp-blink": "^0.1.4"
-  },
-  "engines": {
-    "node": ">= 0.9"
-  },
-  "licenses": [
-    {
-      "type": "MIT",
-      "url": "https://raw.githubusercontent.com/gulpjs/gulp/master/LICENSE"
-    }
-  ]
-}
+{
```

Comment:
These files were causing problems on my machine due to weird line ending stuff.  I had to tell git to ignore the changes to make them go away.